cli/running/running.go

```diff
@@ -376,7 +376,8 @@ func collectInstancesFromAppDir(appDir string, selectedInstName string,
 					InstName:       filepath.Base(appDir),
 					AppDir:         appDir,
 					SingleApp:      true}}, nil
-			} else if instancesScriptsRequired {
+			}
+			if !skipClusterConfig {
```

Comment:
The logic is incorrect here: you are not skip a cluster config load, but skip an error if unable to load a config.